frontend/src/component/common/AutocompleteBox/AutocompleteBox.tsx

```diff
@@ -54,12 +56,80 @@ export const AutocompleteBox = ({
     onChange,
     disabled,
 }: IAutocompleteBoxProps) => {
+    const [placeHolder, setPlaceholder] = useState('Add Segments');
```

Comment:
Since the placeholder is styled to look more "buttonish" it looks very in your face when the cursor appears. We use this state to clear the text out on focus, so that the user can freely focus on the options and search field.